apps/studio/src/lib/db/client.ts

```diff
@@ -82,6 +82,9 @@ export interface DatabaseClient {
   // delete stuff
   dropElement: (elementName: string, typeOfElement: DatabaseElement, schema?: string) => Promise<void>
   truncateElement: (elementName: string, typeOfElement: DatabaseElement, schema?: string) => Promise<void>
+
+  // duplicate table
```

Comment:
Could you possibly also add a ‘duplicateTableSql’ method here (similar to alterTableSql vs alterTable) so users can edit the sql themselves if they like?